rerun_py/rerun_sdk/rerun/sinks.py

```diff
@@ -72,6 +79,9 @@ def save(path: str | pathlib.Path, recording: RecordingStream | None = None) ->
     ----------
     path : str
         The path to save the data to.
+    blueprint: Optional[BlueprintLike]
```

Comment:
Here and below, to make lives easier on ourselves :shrug: 
```suggestion
    blueprint: